src/components/CreateAnalysis/CreateAnalysis.tsx

```diff
@@ -47,15 +46,11 @@ const CreateAnalysis: React.FC = () => {
   // }
 
   return (
-    <Stack>
-      <StackItem>Create a new Predictive analysis</StackItem>
-      <StackItem isFilled>
-        <PatientLookup isOnDashboard={true}></PatientLookup>
-        {/* <Button onClick={submitFile}>Choose File...</Button> */}
-      </StackItem>
-    </Stack>
+    <>
+      <PatientLookup isOnDashboard={true}></PatientLookup>
+      {/* <Button onClick={submitFile}>Choose File...</Button> */}
```

Comment:
Shall we remove this unused code? If yes, also remove `<>` at the same time. @janakitti